blocks/v2-stories-carousel/v2-stories-carousel.js

```diff
@@ -0,0 +1,160 @@
+          <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M8.04885 7.99609C8.04885 5.78695 9.83971 3.99609 12.0488 3.99609C14.2274 3.99609 16.0488 5.77576 16.0488 7.99609C16.0488 10.2052 14.258 11.9961 12.0488 11.9961C9.83971 11.9961 8.04885 10.2052 8.04885 7.99609ZM12.0488 2.99609C9.28742 2.99609 7.04885 5.23467 7.04885 7.99609C7.04885 9.74796 7.94981 11.2894 9.31366 12.1823C6.72461 12.5852 5.31661 13.6165 4.61191 15.0279C4.16935 15.9144 4.03702 16.8943 4.00722 17.8297C3.99226 18.2993 4.00304 18.7677 4.017 19.2149L4.02526 19.4707C4.03701 19.8283 4.04819 20.1686 4.04819 20.4961C4.04819 20.7723 4.27204 20.9961 4.54819 20.9961C4.82433 20.9961 5.04818 20.7723 5.04818 20.4961C5.04818 20.1505 5.03631 19.7897 5.02451 19.4309L5.01651 19.1837C5.00265 18.7395 4.99282 18.2976 5.00671 17.8616C5.03461 16.9859 5.15782 16.1732 5.5066 15.4746C6.17149 14.1429 7.78657 12.9961 12.0001 12.9961C16.2137 12.9961 17.8287 14.1429 18.4935 15.4746C18.8423 16.1732 18.9654 16.9858 18.9933 17.8616C19.0072 18.2976 18.9973 18.7395 18.9835 19.1837L18.9755 19.4307C18.9637 19.7895 18.9518 20.1504 18.9518 20.4961C18.9518 20.7723 19.1756 20.9961 19.4518 20.9961C19.7279 20.9961 19.9518 20.7723 19.9518 20.4961C19.9518 20.1686 19.963 19.8282 19.9747 19.4706L19.9747 19.4705L19.983 19.2149C19.9969 18.7677 20.0077 18.2993 19.9928 17.8298C19.963 16.8943 19.8307 15.9144 19.3882 15.0279C18.6907 13.6308 17.3041 12.6061 14.7649 12.1948C16.1395 11.3038 17.0488 9.75612 17.0488 7.99609C17.0488 5.21643 14.7726 2.99609 12.0488 2.99609Z" fill="currentColor"/>
+          </svg>
+          <address rel="author">${author}</address>
+        </li>
+        <li class="v2-stories-carousel-date">
+          <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M7.5 3C7.77614 3 8 3.22386 8 3.5L8 6C8 6.27614 7.77614 6.5 7.5 6.5C7.22386 6.5 7 6.27614 7 6L7 3.5C7 3.22386 7.22386 3 7.5 3ZM16.5 3C16.7761 3 17 3.22386 17 3.5V6C17 6.27614 16.7761 6.5 16.5 6.5C16.2239 6.5 16 6.27614 16 6V3.5C16 3.22386 16.2239 3 16.5 3ZM3 5.5C3 5.22386 3.22386 5 3.5 5H5.73129C6.00744 5 6.23129 5.22386 6.23129 5.5C6.23129 5.77614 6.00744 6 5.73129 6H4L4 8.99609L20 8.99609V6H18.501C18.2248 6 18.001 5.77614 18.001 5.5C18.001 5.22386 18.2248 5 18.501 5H20.5C20.7761 5 21 5.22386 21 5.5L21 20.5C21 20.7761 20.7761 21 20.5 21L3.5 21C3.22386 21 3 20.7761 3 20.5L3 5.5ZM20 9.99609L4 9.99609L4 20L20 20L20 9.99609ZM8.99927 5.5C8.99927 5.22386 9.22313 5 9.49927 5L14.5027 5C14.7788 5 15.0027 5.22386 15.0027 5.5C15.0027 5.77614 14.7788 6 14.5027 6L9.49927 6C9.22313 6 8.99927 5.77614 8.99927 5.5Z" fill="currentColor"/>
+          </svg>
+          <time datetime="${date}" pubdate="pubdate">${date.toLocaleDateString()}</time>
+        </li>
+      </ul>
+      <a href="${path}" class="v2-stories-carousel-cta">
```

Comment:
Here you should make use of the generic styling. See [.redesign-v2 a.button.primary](https://github.com/Netcentric/vg-volvotrucks-us-rd/blob/c72a665a6076a587e72f09c7edb86a627fdb9344/styles/styles.css#L948)

Even though I like having motion in this place, please remove it for now.